velox/exec/tests/AggregationFuzzerTest.cpp

```diff
@@ -40,6 +40,7 @@ DEFINE_string(
 int main(int argc, char** argv) {
   facebook::velox::aggregate::prestosql::registerAllAggregateFunctions();
   facebook::velox::functions::prestosql::registerAllScalarFunctions();
+  facebook::velox::functions::prestosql::registerInternalFunctions();
```

Comment:
Why can't we register 'canonicalize' function directly?

```
VELOX_REGISTER_VECTOR_FUNCTION(
      udf_$internal$canonicalize, "$internal$canonicalize");
```

Reply:
I came up with the linker error when I used this macro directly here, even though I added the `extern` and `velox_functions_prestosql_impl`.
```Git
+++ b/velox/exec/tests/AggregationFuzzerTest.cpp
+#include "velox/expression/VectorFunction.h"
+extern void registerVectorFunction_udf_$internal$canonicalize(const std::string& name);
 int main(int argc, char** argv) {
   facebook::velox::aggregate::prestosql::registerAllAggregateFunctions();
   facebook::velox::functions::prestosql::registerAllScalarFunctions();
-  facebook::velox::functions::prestosql::registerInternalFunctions();
+  //facebook::velox::functions::prestosql::registerInternalFunctions();
+  VELOX_REGISTER_VECTOR_FUNCTION(
+      udf_$internal$canonicalize, "$internal$canonicalize");

 +++ b/velox/exec/tests/CMakeLists.txt
@@ -197,6 +197,7 @@ add_executable(velox_aggregation_fuzzer_test AggregationFuzzerTest.cpp)

 target_link_libraries(
   velox_aggregation_fuzzer_test
+        velox_functions_prestosql_impl
```

Reply:
Got it. Then let's leave it as is. Thanks.